lib/lookup.json

```diff
@@ -245,6 +239,15 @@
     "prefix": "v",
     "maintainers": "juliangruber"
   },
+  "jest": {
+    "prefix": "v",
+    "maintainers": ["cpojer", "scotthovestadt", "SimenB", "thymikee", "jeysal"],
+    "yarn": true,
+    "scripts": ["build:js", "test-ci-partial"],
```

Comment:
this ran tests locally, but seems like CI skipped it? very odd...

![image](https://user-images.githubusercontent.com/1404810/96614436-6b88fe80-1300-11eb-9bd3-7725a76fa856.png)

I'll have to try tomorrow

Reply:
no idea about this - the tests were ran on mac, but timed out.